index.html

```diff
@@ -0,0 +1,751 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="CSS, styles, styleSUP,Supminn,SupTECH, Web developer">
+    <meta name="description" content="styleSUP CSS Component Library">
+    <meta name="author" content="Supriya M">
+    <meta name="Copyright" content="SupTECH">
+    <link rel="icon" href="./images/logo.png" type="image/png">
+    <link rel="stylesheet" href="./styles/components.css">
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+    <title>StyleSUP | Docs</title>
+</head>
+
+<body>
+    <div class="grid-body">
+        <!-- Navigation -->
+        <nav id="navigation" class="nav">
+            <div class="nav-logo">
+                <img class="img-rd" src="./images/logo.png" alt="logo" />
+                <span class="nav-header">StyleSUP</span>
+            </div>
+            <div class="social-media">
+                <a class="link" target="_blank" href="https://github.com/supminn">
+                    <img class="media-logo" src="./images/github.svg" alt="github" />
+                </a>
+                <a class="link" target="_blank" href="https://twitter.com/supminn">
+                    <img class="media-logo" src="./images/twitter.svg" alt="twitter" />
+                </a>
+            </div>
+            <span id="btn-navigation" class="nav-btn"><i class="fas fa-chevron-circle-down fa-lg"></i></span>
+        </nav>
+        <!-- Left Panel - Action -->
+        <aside class="left-pannel">
+            <ul id="list-nav-main" class="nav-list nav-list-hide">
```

Comment:
styling with id rather than classnames. Why ?

Reply:
This id: "list-nav-main" is being used within script.js to toggle the left-panel options on mobile view.

Reply:
Supriya, I am saying that you did styling of this with id. Toggling is correct with id. style ul with classname and not id.
This part:

@media screen and (min-width: 768px)
#list-nav-main {
    position: sticky;
    top: 13%;
    left: 1%;
    width: max-content;
    margin: 1rem;
    border: 1px solid var(--primaryBg);
    border-radius: 1rem;
    background-color: var(--theme50);
}